app.py

```diff
@@ -129,25 +242,10 @@ def render_and_submit_path(self, path, fields, first_frame, last_frame, sg_publi
         output_path_template = self.get_template("movie_path_template")
         output_path = output_path_template.apply_fields(fields)
 
-        # Get the name for the Version entity
-        version_template = self.get_template("sg_version_name_template")
-        version_name = None
-        if version_template:
-            version_name = version_template.apply_fields(fields)
-
-        fields["description"] = comment
-
-        # Render and Submit
-        progress_cb(20, "Rendering movie")
-        renderer = tk_multi_reviewsubmission.Renderer()
-        renderer.render_movie_in_nuke(path, 
-                                      output_path, 
-                                      width, height, 
-                                      first_frame, last_frame,
-                                      fields.get("version", 0), 
-                                      fields.get("name", "Unnamed"),
-                                      color_space, fields)
+        if output_path not in processed_paths:
+            raise Exception("tk-multi-reviewsubmission not configured to render movie!")
```

Comment:
Please use a more specific Exception

Reply:
How will specific Exceptions help? Since these modules are can't be imported directly anyways.

Reply:
The Exception class still provides useful information in the traceback to communicate the developer's intent. Imagine a TD a year from now with no prior knowledge of this tool having to debug an issue. If that was you, what information would you want here?

Reply:
Makes sense, will add the new Exceptions.